### DISCUSSION TITLE
Toot impressions?

### DISCUSSION THREAD

**Discussion by hershkoy** [mastodon/mastodon#25709] [General]
Newbie here.

I just joined two days ago, and overall I like what I see. Kodos for the great work, really.

Question: Do you think that Mastodon will have an impression counter for each toot, similar to what exists in Twitter?
It doesn't have to be public, can be visible only to the owner

Just wondering if it was considered.

**Comment by trwnh** [comment]
This is not possible because posts do not flow through any central point or tracker.
